### PR TITLE
[SRVCOM-1740]: Add 1.23.0 release notes

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
OCP 4.6+
OSD 4
ROSA

Issues:
https://issues.redhat.com/browse/SRVCOM-1740

Link to docs preview:
- OCP: http://file.rdu.redhat.com/abrennan/230622-OCP/SRVCOM-1740/serverless/serverless-release-notes.html
- OSD: http://file.rdu.redhat.com/abrennan/210622-OSD/SRVCOM-1740/serverless/serverless-release-notes.html
- ROSA: http://file.rdu.redhat.com/abrennan/230622-ROSA/SRVCOM-1740/serverless/serverless-release-notes.html

### CODE REVIEW

File: serverless/admin_guide/serverless-ha.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ High availability (HA) functionality is available by default on {ServerlessProdu
 
 * Knative Serving: `activator`, `autoscaler`, `autoscaler-hpa`, `controller`, `webhook`, `kourier-control`, `kourier-gateway`.
 * Knative Eventing: `eventing-controller`, `eventing-webhook`, `imc-controller`, `imc-dispatcher`, `mt-broker-controller`, `sugar-controller`.
-* Knative Kafka: `kafka-ch-controller`, `kafka-controller-manager`, `kafka-webhook`.
+* Knative Kafka: `kafka-controller`, `kafka-webhook-eventing`.
 
 These components are configured with two replicas by default.
```
Comment (abrennan89) [inline serverless/admin_guide/serverless-ha.adoc:23]: This whole section is actually updated by a different PR, so this isn't needed in this PR. Will add a link later.

Comment (abrennan89) [reply serverless/admin_guide/serverless-ha.adoc:23]: Link: https://github.com/openshift/openshift-docs/pull/46468

PR is ready to merge, can be merged after this one.

File: modules/serverless-rn-1-23-0.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      limits:
+        cpu: 1000m
+        memory: 1000Mi
+----
+
+* If you are using link:https://buildpacks.io/[Cloud Native Buildpacks] as the local build strategy for a function, `kn func` is unable to automatically start podman or use an SSH tunnel to a remote daemon. The workaround for these issues is to have a Docker or podman daemon already running on the local development computer before deploying a function.
+
+* On-cluster function builds currently fail for Quarkus and Golang runtimes. They work correctly for Node, Typescript, Python, and Springboot runtimes.
+
+* There is currently a known issue when using a Kafka channel-backed, multi-tenant broker under conditions where the throughput of the Kafka channel outpaces the limit of the container as set by kubernetes.
```
Comment (mgencur) [inline modules/serverless-rn-1-23-0.adoc:67]: I'm curious, which issue is this one in JIRA? Thanks
Otherwise LGTM

Comment (abrennan89) [reply modules/serverless-rn-1-23-0.adoc:67]: TBH I can't remember; I think part of the issue is folks adding and removing issues from the links in the release notes PR, it's very hard to keep track.
I think it was from @pierDipi maybe, so he might remember?

File: modules/serverless-rn-1-23-0.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+not identified yet
+
+[id="fixed-issues-1-23-0_{context}"]
+== Fixed issues
+////
+
+[id="known-issues-1-23-0_{context}"]
+== Known issues
+
+* The Federal Information Processing Standards (FIPS) mode is disabled for Kafka broker, Kafka source, and Kafka sink.
```
Comment (abrennan89) [inline modules/serverless-rn-1-23-0.adoc:37]: No related Jira, asked for in PM call

File: modules/serverless-rn-1-23-0.adoc [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+* If you delete a namespace that includes a Kafka broker, the namespace finalizer may fail to be removed if the broker's `auth.secret.ref.name` secret is deleted before the broker.
+
+* Running {ServerlessProductName} with a large number of Knative services can cause Knative activator pods to run close to their default memory limits of 600MB. These pods might be restarted if memory consumption reaches this limit. Requests and limits for the activator deployment can be configured by modifying the `KnativeServing` custom resource:
```
Comment (abrennan89) [inline modules/serverless-rn-1-23-0.adoc:41]: https://issues.redhat.com/browse/SRVKS-936

File: modules/serverless-rn-1-23-0.adoc [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    - container: activator
+      requests:
+        cpu: 300m
+        memory: 60Mi
+      limits:
+        cpu: 1000m
+        memory: 1000Mi
+----
+
+* If you are using link:https://buildpacks.io/[Cloud Native Buildpacks] as the local build strategy for a function, `kn func` is unable to automatically start podman or use an SSH tunnel to a remote daemon. The workaround for these issues is to have a Docker or podman daemon already running on the local development computer before deploying a function.
```
Comment (abrennan89) [inline modules/serverless-rn-1-23-0.adoc:63]: https://issues.redhat.com/browse/SRVOCF-430

File: modules/serverless-rn-1-23-0.adoc [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+* On-cluster function builds currently fail for Quarkus and Golang runtimes. They work correctly for Node, Typescript, Python, and Springboot runtimes.
```
Comment (abrennan89) [inline modules/serverless-rn-1-23-0.adoc:65]: https://issues.redhat.com/browse/SRVOCF-429

File: modules/serverless-rn-1-23-0.adoc (hunk 2 above)
Comment (abrennan89) [reply modules/serverless-rn-1-23-0.adoc:67]: If we can't identify which Jira this was for I'll remove it.

Comment (abrennan89) [reply modules/serverless-rn-1-23-0.adoc:67]: Asked in Slack https://coreos.slack.com/archives/CD87JDUB0/p1655911061891679

Comment (mgencur) [reply modules/serverless-rn-1-23-0.adoc:67]: Cool. Thanks! So let's remove it.

**Comment by mgencur** [comment]
LGTM

**Review by maxwelldb — APPROVED** [review]

**Pull request merged**